src/lib/containers/FacetsPlotNav.tsx

```diff
@@ -1,317 +0,0 @@
-import Plotly from 'plotly.js-basic-dist'
```

Comment:
I think this was my original prototype for the facet plot-based navigation, was not referenced outside of a test.